code/src/App.js

```diff
@@ -1,9 +1,78 @@
-import React from 'react'
+import React, {useEffect, useState} from 'react';
+
+import ThoughtsForm from './components/ThoughtsForm';
+import ThoughtsContainer from './components/ThoughtsContainer';
+import {API_URL, LIKE_URL} from './reusable/urls';
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+  const [newThought, setNewThought] = useState('');
+
+  useEffect (() => {
+    fetchThoughts();
+  }, []);
+
+  const fetchThoughts = () => {
+    fetch(API_URL)
+    .then(response => response.json())
+    .then(thoughts => setThoughts(thoughts))
+    .catch(err => console.error(err));
+    
```

Comment:
Maybe remove the empty line here :)

Reply:
Fixed it! =)

Reply:
And console.log from the finished project 😅

Reply:
Done cleaning console(log):s =)